modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C

```diff
@@ -137,9 +137,12 @@ INSFVRhieChowInterpolator::INSFVRhieChowInterpolator(const InputParameters & par
     if (blockIDs() != var.blockIDs())
       mooseError("Block restriction of interpolator user object '",
                  this->name(),
-                 "' doesn't match the block restriction of variable '",
+                 "' (",
+                 std::accumulate(blocks().begin(), blocks().end(), std::string{}),
```

Comment:
Isn't this just going to concatenate block names without any separating whitespace or punctuation?

Reply:
cant remember lemme check

Reply:
good catch! not sure what I was thinking